packages/ui5-application-writer/templates/optional/typescript/package.json

```diff
@@ -0,0 +1,19 @@
+{
+    "scripts": {
+        "ts-typecheck": "tsc --noEmit"
```

Comment:
This script on its own is not that useful. it should be integrated into build and or preview scripts. 

When I run `npm run ts-typecheck` in `/packages/fiori-freestyle-writer/test/test-output/listDetailTemplate/listdetail-ts`.

```
Found 124 errors in 8 files.

Errors  Files
     2  webapp/Component.ts:45
     8  webapp/controller/App.controller.ts:10
     8  webapp/controller/BaseController.ts:5
    48  webapp/controller/Detail.controller.ts:4
     8  webapp/controller/ErrorHandler.ts:22
    35  webapp/controller/List.controller.ts:11
    12  webapp/controller/ListSelector.ts:21
     3  webapp/controller/NotFound.controller.ts:8
````
All these errors should be fixed (and in other floorplans) to be a true TS project.

Reply:
@devinea  Would it be ok for you to accept these errors for now, and we merge this new minor version. I will start right away with a new branch on creating a patch fixing these errors and some others issues I saw in the js templates.
I thought about fixing these here right away but the PR is big enough already.

Reply:
Resolving the conversation as discussed.